source/renderer/app/utils/formatters.js

```diff
@@ -19,7 +13,30 @@ export const formattedWalletAmount = (
   let formattedAmount = '';
   if (long) {
```

Comment:
Please use ternary operator condition

Reply:
@DeeJayElly can you please fix this one?

Reply:
Done